lagom-eventstore-example/resources/lagom-event-store-example-greetings.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
This is humna-readable but it's more like a pseudo-binary file meant for IPython/Jupyter Notebook to parse. You can try to read and parse but it's not tested at the moment so it's not worth the effort at the moment.